app/helpers/runnables_helper.rb

```diff
@@ -61,8 +61,15 @@ def student_run_buttons(offering,opts={})
   end
 
   def runnable_type_label(component)
+
     runnable = component.is_a?(Portal::Offering) ? component.runnable : component
+    
+    if runnable.respond_to?(:material_type) && runnable.material_type == 'Interactive'
+        return I18n.t(runnable.material_type.to_s.underscore.to_sym).titleize
+    end
+
```

Comment:
Unless you see a problem with it. I think it would be better to make this consistent for all ExternalActivities.  Since all ExternalActivities now have a material_type set.  Then the conditional could be:
`if runnable.is_a?(ExternalActivity)`
And then it would get the material_type and look it up the localization.

The block below that is checking for a template_type could be dropped, since this only applies to ExternalActivities.